patches/chromium/os-16623_add_setopacity_and_downgrade_to_zwp_alpha_compositing_v1.patch

```diff
@@ -0,0 +1,409 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Tariq Bashir <120014322+t-bashir-bs@users.noreply.github.com>
+Date: Thu, 18 Jan 2024 13:27:03 +0000
+Subject: OS-16623 add setopacity and downgrade to zwp_alpha_compositing_v1
+
+This patch downgrades the alpha_compositing protocol from zcr_alpha_compositing_v1
+to zwp_alpha_compositing_v1 to match the protocol version used by the BS
+wayland/weston compositor. This patch also adds a setopacity method to the
+wayland_toplevel_window class to allow the opacity of the window to be set.
```

Comment:
Is this a revert of upstream commit? If that is the case, can you put the commit hash, repo path and original commit comment here.

Reply:
I had a look at Chromium git. 
This is not a revert of a single change. The protocol file downgrade is a partial revert of this commit from 7 years ago:
https://chromium.googlesource.com/chromium/src/+/ef576bfe8a8feee1b27213e301907758c7bcd0c8
However, I reverted by copying the contents of our alpha-compositing-unstable-v1.xml protocol file and not what was in the Chromium one before.
The changes to other ozone files are not a revert because they were added much later and already were using zcr_alpha_compositing_v1:
https://chromium.googlesource.com/chromium/src/+/c3b74f1338e2f2daffbd71f04ae176f1975e7c99%5E%21/#F3